lib/Params.ml

```diff
@@ -250,7 +248,11 @@ module Exp = struct
             match ctx_is_apply_and_exp_is_arg ~ctx ~ctx0 with
             | Some (Nolabel, fun_exp, is_last_arg) ->
                 if begins_line fun_exp.pexp_loc then
-                  if is_last_arg then 5 else 3
+                  if is_last_arg then 5
+                  else
+                    (* TODO Is this branch dead ? Changing the value doesn't
```

Comment:
I couldn't find any example when running `test_branch` with `profile=janestreet` either. This seems to indicate that it is dead code, thanks for looking at that!

Reply:
i was fiddling at the value to try and "fix" something